### PR TITLE
Add local shape file component

### PR DESCRIPTION
**Pull request by kappu72**

### CODE REVIEW

File: web/client/plugins/toolbar/index.js [hunk 1]
```diff
@@ -157,6 +159,15 @@ module.exports = (context) => ([{
     title: "snapshot.title",
     exclusive: true
 }, {
+    name: 'shapefile',
+    panel: ShapeFile,
+    help: <Message msgId="helptexts.snapshot"/>,
```
Comment (mbarto) [inline web/client/plugins/toolbar/index.js]: helptexts.shapefile here

File: web/client/translations/data.en-US [hunk 2]
```diff
@@ -317,6 +317,13 @@
             "tooltip": "Save a snapshot of the map.",
             "googleBingError": "Googole and Bing layers aren't available for the snapshot, due to copyright restrictions.",
             "downloadingSnapshots": "pending snapshot generation"
+        },
+        "shapefile": {
+            "title": "Add Local Shape File",
+            "tooltip": "Add a local shape file to the map.",
+            "placeholder": "Drop or click to import a local Shape",
```
Comment (offtherailz) [inline web/client/translations/data.en-US]: Drop your file here or click to import a local shape file.

File: web/client/translations/data.it-IT [hunk 3]
```diff
@@ -318,6 +318,13 @@
             "tooltip": "Salva un' istantanea della mappa.",
             "googleBingError": "Istantanea non disponibile per le restrizioni all' utilizzo dei layer Google e Bing.",
             "downloadingSnapshots": "generazioni di istantanee in corso..."
+        },
+        "shapefile": {
+            "title": "Carica Shape File",
+            "tooltip": "Aggiungi uno shape file alla mappa.",
+            "placeholder": "Drop or click per importare uno Shapefile",
```
Comment (offtherailz) [inline web/client/translations/data.it-IT]: Trascina qui il file o fai click per selezionare lo shape file da importare.

**Comment by simboss** [comment]
Where is the issue for the task of implementing this feature?
We must have an issue for each PR so that when we can track work done using issues.
The PR should reference the issue.

If there is no associated issues go ahead and open one.

**Pull request merged**